package.json

```diff
@@ -30,6 +30,7 @@
     "twilio": "^3.62.0"
   },
   "devDependencies": {
+    "express": "^4.17.1",
```

Comment:
:eyes:

Reply:
Yes it was needed by Twilio in dev mod, I'm not sure why :facepalm: